source/includes/_users.md

```diff
@@ -471,7 +475,11 @@ Argument | Type | Description
 query | `string` | Query to use for searching.
```

Comment:
Could you also make this text better? 
Like: "The search query. `first_name`, `last_name` and `id`.